ibex_configs.yaml

```diff
@@ -150,4 +157,58 @@ experimental-branch-predictor:
   PMPNumRegions            : 4
   SecureIbex               : 0
   ICacheScramble           : 0
+  XInterface               : 0
 
+# Small with X-Interface
+small-cvxif:
+  RV32E                    : 0
+  RV32M                    : "ibex_pkg::RV32MFast"
+  RV32B                    : "ibex_pkg::RV32BNone"
+  RegFile                  : "ibex_pkg::RegFileFF"
+  BranchTargetALU          : 0
+  WritebackStage           : 0
+  ICache                   : 0
+  ICacheECC                : 0
+  BranchPredictor          : 0
+  PMPEnable                : 0
+  PMPGranularity           : 0
+  PMPNumRegions            : 4
+  SecureIbex               : 0
+  ICacheScramble           : 0
+  XInterface               : 1
+
+# Opentitan with X-Interface
+opentitan-cvxif:
+  RV32E                    : 0
+  RV32M                    : "ibex_pkg::RV32MSingleCycle"
+  RV32B                    : "ibex_pkg::RV32BOTEarlGrey"
+  RegFile                  : "ibex_pkg::RegFileFF"
+  BranchTargetALU          : 1
+  WritebackStage           : 1
+  ICache                   : 1
+  ICacheECC                : 1
+  BranchPredictor          : 0
+  PMPEnable                : 1
+  PMPGranularity           : 0
+  PMPNumRegions            : 16
+  SecureIbex               : 1
+  ICacheScramble           : 0
+  XInterface               : 1
+
+# experimental-maxperf-pmp config with X-Interface
+experimental-maxperf-pmp-bmfull-cvxif:
+  RV32E                    : 0
+  RV32M                    : "ibex_pkg::RV32MSingleCycle"
+  RV32B                    : "ibex_pkg::RV32BFull"
+  RegFile                  : "ibex_pkg::RegFileFF"
+  BranchTargetALU          : 1
+  WritebackStage           : 1
+  ICache                   : 0
+  ICacheECC                : 0
+  BranchPredictor          : 0
+  PMPEnable                : 1
+  PMPGranularity           : 0
+  PMPNumRegions            : 16
+  SecureIbex               : 0
+  ICacheScramble           : 0
+  XInterface               : 1
```

Comment:
Missing newline at the end of the file.

Reply:
> This looks good to me, well done!
> 
> There is one missing newline in the configs.yaml file that you should remove. After that we can merge this.
> 
> On the CI testing: the idea of the CI testing is primarily to verify that we don't break existing functionality. Once we have the CVXIF fully implemented and a co-processor attached, we can also enable CI testing of that. Up to this point, it's best if you the test the CVXIF configurations locally on your machine. There you also have maximum visibility for debugging. Once you know stuff is working at your end, you can update your PR to trigger CI. But CI is not meant as primary tool for verifying/debugging your work. :-)

Hello,
I have add the newline. I have also make a small change at some .core files.
```
  XInterface:
    datatype: int
    default: 0
    paramtype: vlogparam
    description: "Enables the RISC-V Extension Interface (EXPERIMENTAL) [0/1]"
```
I changed the default value from 1 to 0, which I think is cleaner version.